apple/bundling/plist_support.bzl

```diff
@@ -40,7 +40,7 @@ def _extract_provisioning_plist_command(ctx, provisioning_profile):
     # be somewhat preferable to one that implies failure. Regardless, we capture
```

Comment:
can you update this comment? it's not true after this change

Reply:
updated